store/localstore/goleveldb/goleveldb.go

```diff
@@ -44,9 +44,8 @@ type db struct {
 func (d *db) Get(key []byte) ([]byte, error) {
 	v, err := d.DB.Get(key, nil)
 	if err == leveldb.ErrNotFound {
-		return nil, nil
+		return nil, errors.Wrap(err, engine.ErrNotFound)
```

Comment:
Maybe we should return `errors.Trace(engine.ErrNotFound)` at least? Since Seek returns this error too.